content/en/docs/concepts/packages/_index.md

```diff
@@ -1,11 +1,279 @@
+
+CUE files may define a package name at the top of their file.
+CUE uses this to determine which files belong together.
+If the `cue` tool is told to load the files for a specific directory,
+for instance:
+
+    cue eval ./mypkg
+
+it will only look files with such a clause and ignore files without it.
+
+If the package name within the directory is not unique, `cue` needs to
+know the name of the package as well
+
+    cue eval -p pkgname ./mypkg
+
+If no module is defined, it will just load the files in this directory.
+If a module is defined, it will _also_ load all files with the same
+package name in its ancestor directories up till the module root.
+As we will see below,
+this strategy allows for defining organization-wide schemas and policies.
+
+
+### Import path
+
+Each package is identified by a globally unique name, called its _import path_.
+The import path consists of a unique location identifier
+followed by a colon (`:`) and its package name.
+
+    k8s.io/api/core/v1:v1
+
+If the basename of the path and the package name are the same, the latter
+can be omitted.
+
+    k8s.io/api/core/v1
+
+The unique location identifier consists of a domain name followed by a path.
+
+Modules themselves also have a unique location identifier.
+A package inside a module can import another package from this same module
+by using the following import path:
+
+    <module identifier>/<relative position of package within module>:<package name>
+
+So suppose our module is identified as `example.com/pkg` and a package
+located at `schemas/trains` and has the package name `track`,
+then other packages can import this packages as:
+
+    import "example.com/pkg/schemas/trains:track"
+
+Putting it all together:
+```
+root                    // must contain:
+|-- cue.mod
+|   |-- module.cue      // module: "example.com/pkg"
+|-- schemas
+|   |-- trains
+|   |   |-- track.cue   // package track
+...
+|-- data.cue            // import "example.com/pkg/schemas/trains:track"
+```
+
+The relative position may not be within the `cue.mod` directory.
+
+<!-- TODO: about how to resolve these names into concrete locations. -->
+
+### Location on disk
+
+A `.cue` file can import a package by specifying its import path
+with the import statement. For instance,
+
+    import (
+        "list"
+
+        "example.com/path/to/package"
+    )
+
+Packages for which the first path component is not a fully qualified
+domain name are builtin packages and are not stored on disk.
+For other packages, CUE determines the location on disk as follows:
+
+1. If a module identifier is defined and is a prefix of the import path,
+   the package is located at the corresponding location relative to the
+   module root.
+2. Otherwise, the package contents looked up in
+   the `cue.mod/pkg`, `cue.mod/gen`, and `cue.mod/usr` subdirectores.
+
+In Step 2, an import path may match more than one directory.
+In that case, the contents of _all_ matched directories are used to build the
```

Comment:
Would using the word unification lead to more precision?

"In that case, the contents of _all_ matched directories are unified.
Virtually, these directories should be seen as a single directory tree."

Reply:
It would certainly be more precise. I'm generally trying to avoid the word unification in contexts where the user may not yet have learned about it.